fedimint-server/src/consensus/aleph_bft/finalization_handler.rs

```diff
@@ -25,6 +33,9 @@ impl aleph_bft::FinalizationHandler<UnitData> for FinalizationHandler {
     }
 
     fn unit_finalized(&mut self, creator: NodeIndex, round: Round, data: Option<UnitData>) {
+        let mut count = *self.unit_count_sender.borrow();
+        count += 1;
+        self.unit_count_sender.send(count).ok();
```

Comment:
You may want to use [`send_modify`](https://docs.rs/tokio/latest/tokio/sync/watch/struct.Sender.html#method.send_modify) instead.